analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/InitializationVectorShouldBeRandom.cs

```diff
@@ -124,8 +146,8 @@ public void CollectionInitializer()
         List<byte> list = new List<byte> { Rnd() };
         using (AesCryptoServiceProvider aes = new AesCryptoServiceProvider())
         {
-            ICryptoTransform encryptor = aes.CreateEncryptor(aes.Key, listWithConstant.ToArray()); // FIXME Non-compliant
-            encryptor = aes.CreateEncryptor(aes.Key, list.ToArray()); // FIXME Non-compliant
+            ICryptoTransform encryptor = aes.CreateEncryptor(aes.Key, listWithConstant.ToArray()); // FN
+            encryptor = aes.CreateEncryptor(aes.Key, list.ToArray());
```

Comment:
Are we considering this Compliant because we don't know what `Rnd()` does and assume it to be safe? Do you know why the old implementation had it as Noncompliant?

Reply:
It's related to the discussion we had with AppSec.
That we should raise in this case, since on one side we don't know, and on the other initializing like seems sketchy.

Reply:
Then it should be marked as Noncomplient, no?